source/kit/components/ui/collapsingNavbar.js

```diff
@@ -79,7 +84,9 @@
             stickyBars.css('top', '75px');
```

Comment:
Hm. Should be ok, but I'd prefer this to be a default setting in the config object I've mentioned earlier. Same thing applies to timeouts, etc. What do you think? (#1stworldproblems again, I'm totally aware of that)